### ISSUE TITLE
instead of error messages we always get `fault response : struct mapped to type Fault`

### ISSUE THREAD

**Issue by lorddev** [LordVeovis/xmlrpc#9]
Since updating to this .NET Core project from the regular Cook .NET one, instead of error messages we always get `fault response : struct mapped to type Fault`

**Comment by LordVeovis** [comment]
hi, can you specify the used version of the library?

**Comment by lorddev** [comment]
We forked from your repo in October https://github.com/lorddev/xmlrpc

**Comment by lorddev** [comment]
@LordVeovis here is the full error:
> fault response contains struct value with missing non-optional members: FaultCode FaultString [fault response : struct mapped to type Fault]

**Comment by LordVeovis** [comment]
I integrated #8 some times ago, which fix this error.
Mainly when there is an exception from one endpoint, the error is transmitted using the Fault struct.
When I modified the Cook's lib to be compatible with .netcore, I did change the case on the Fault properties (FaultCode instead of faultCode, and FaultString instead of faultString).

If you take the last version (1.1.1+) on nuget or merge the update, this should resolve the issue.
